spring-boot/admin/src/main/resources/application-github.yml

```diff
@@ -0,0 +1,24 @@
+spring:
```

Comment:
Since GitHub is used by default and using KeyRock is just a special case/feature, shouldn't this configuration be contained in `application.yml`?

Reply:
No, the issue will be that until now I didn't find a way that if we deploy Keyrock, then the Github config needs to be disabled.
So what would happen, is that actually CoatRack would be deployed with both Github and Keyrock as identity managers, as the big change of the spring security 5 is that you can add as much Idms as you wish to your web application.
I hope I was clear explaing this, if not we can do a short TelCo